src/web/client/extension.ts

```diff
@@ -22,7 +22,6 @@ export function activate(context: vscode.ExtensionContext): void {
     context.subscriptions.push(_telemetry);
 
     setTelemetryReporter(_telemetry);
-    sendInfoTelemetry("Start");
```

Comment:
add the Start event back. This is required for our telemetry pipeline to work. 'Start' means 'session started'.
You should add on the user's pac CLI user id, as is done in the desktop client's extension.